x/wasm/keeper/addresses_test.go

```diff
@@ -3,9 +3,9 @@ package keeper
 import (
 	"encoding/json"
 	"fmt"
-	tmbytes "github.com/cometbft/cometbft/libs/bytes"
 	"testing"
 
+	tmbytes "github.com/cometbft/cometbft/libs/bytes"
```

Comment:
This change should not be in here. Please run `make format`

Reply:
This is the result of `make format`. It's in a separate commit. Feel free to kick it out if needed